app/src/main/res/values-fi/strings.xml

```diff
@@ -0,0 +1,44 @@
+<resources>
+    <string name="main_menu_game_actions">Pelitoiminnot</string>
+    <string name="main_menu_onscreen_input_setup">Ruudulla olevien syötteiden asetukset</string>
+    <string name="load_game_from_file_explanation">Voit ladata Virtual Boy -pelien ROM-muisteja ja ladata ne tänne pelataksesi niitä.\nVoit löytää ilmaisia ROM-muisteja Planet Virtual Boysta, osoitteesta https://virtual-boy.com/homebrew/.</string>
+    <string name="load_game_no_recent_games">Et ole vielä pelannut yhtään peliä. Kokeile yhtä!</string>
+    <string name="load_game_no_bundled_games">Ei pakattuja pelejä</string>
+    <string name="load_game_created_by">Luonut %1$s</string>
+    <string name="game_menu_resume_game">Jatka peliä</string>
+    <string name="game_menu_save_state">Tallennustila</string>
+    <string name="game_menu_state_slot">Virran tilan paikka</string>
+    <string name="game_menu_state_slot_empty">Tyhjä</string>
+    <string name="game_menu_close_game">Sulje peli</string>
+    <string name="onscreen_input_menu_on">Käytä virtuaalista pelikonsolia</string>
+    <string name="onscreen_input_menu_preview">Esikatselu</string>
+    <string name="controller_menu_manage">Hallitse ohjaimia</string>
+    <string name="controller_menu_new_controller">Uusi ohjain</string>
+    <string name="controller_menu_create">Luo</string>
+    <string name="controller_menu_choose_rename">Valitse uudelleen nimettävä ohjain...</string>
+    <string name="controller_menu_choose_delete">Valitse poistettava ohjain...</string>
+    <string name="input_menu_put_mapping">Paina mitä tahansa ohjainta, jotta se voidaan liittää tähän syötteeseen. Paina pitkään kartoittaaksesi useita ohjaimia.</string>
+    <string name="input_menu_add_mapping">Paina mitä tahansa ohjainta lisätäksesi toisen kartoituksen tälle syötteelle.</string>
+    <string name="video_menu_screen_zoom_percent">Ruudun zoomaus (prosenttia)</string>
+    <string name="video_menu_vertical_offset">Vertikaalinen offset</string>
+    <string name="video_menu_mode">Näyttötila</string>
+    <string name="video_mode_anaglyph_summary">Anaglyph (kaksivärinen)</string>
+    <string name="video_mode_anaglyph_description">Toimii useimpien 3D-lasien kanssa</string>
+    <string name="video_mode_cardboard_description">Toimii Google Cardboardin tai Daydreamin kanssa</string>
+    <string name="video_mode_mono_left_summary">Vasen silmä vain</string>
+    <string name="video_mode_mono_right_summary">Vain oikea silmä</string>
```

Comment:
Since the English values are "Left eye only" and "Right eye only", shouldn't these be symmetric too? Like "Vain vasen silmä" and "Van oikea silmä"?